Add info about EasyCLA 👩‍⚖️ (#98)
tektoncd/community: process.md

* Try to record the meeting, and post a link to the recording.
* If the meeting will be recurring, or have a large enough audience, use a poll to allow participants to vote on
  potential times.

## CLA

To contribute to repos in tektoncd you need to be authorized to contributed under the CDF Contributor's License
Agreement (CLA) which is managed by EasyCLA via https://project.lfcla.com/.

> That contributors have been authorized is managed via the CommunityBridge EasyCLA GitHub app. The first time you

Reviewer: ```suggestion
Contributors are authorized and managed via the CommunityBridge EasyCLA GitHub app. The first time you
```

Author: Thanks @popcor255 !!
